spec/lib/vcr/errors_spec.rb

```diff
@@ -102,7 +110,15 @@ def request_with(values)
           end
         end
 
-        it 'mentions :allow_playback_repeats if the current cassette has a used matching interaction' do
+        it 'does not mention the :once or :none record modes if using the :new_episodes record mode at least in one cassette' do
+          VCR.use_cassette('example', :record => :new_episodes) do
+            VCR.use_cassette('sample') do
+              expect(message).not_to include('current record mode (:once)', 'current record mode (:none)')
```

Comment:
Line is too long. [106/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.